config/src/generator.rs

```diff
@@ -50,6 +50,12 @@ pub fn validator_swarm(
         network.seeds = seeds.clone();
     }
 
+    nodes.sort_by(|a, b| {
```

Comment:
Why is this needed?

Reply:
I wanted the debug assert to ensure that they were sorted by move, but in some cases we generate a validator set independent of genesis -- and then that debug assert will fail.